CHANGELOG.md

```diff
@@ -0,0 +1,104 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Looking at the amendment made to `setup.py` in this same PR, shouldn't this be `v0.1.0` ?

Reply:
Was thinking that should be done off `dev`... as the `setup.py` change should have been too - doh!